test/support/rails_app.rb

```diff
@@ -5,6 +5,7 @@
 class RailsApp < Rails::Application
   config.logger = Rails.logger = Logger.new(nil)
   config.secret_key_base = "foo"
+  config.middleware.use ActionDispatch::Session::CookieStore, key: '_namespace_key'
```

Comment:
Just curious - was this a new requirement?

Reply:
I changed it now to 

```
  config.session_store :cookie_store, key: '_anything'
```

but yeah, we need to set `session_store` as `:cookie_store` since we're using session in sinatra app